fedimint-server/src/consensus/engine.rs

```diff
@@ -393,13 +396,16 @@ impl ConsensusEngine {
                         if self.keychain.verify(&header, &signature, to_node_index(peer)){
                             signatures.insert(peer, signature);
                         } else {
-                            warn!(target: LOG_CONSENSUS, "Received invalid signature from peer {peer}");
+                            error!(target: LOG_CONSENSUS, "Consensus Failure: invalid header signature from {peer}");
```

Comment:
Could we also print all our items here? I suppose all guardians would get that error and then we could compare?

Reply:
Hmm... Actually, right...

If all guardians print all items whenever they receive a wrong signature, all prefixed if their ID, then it should be relatively easy to binary search the outstanding item.

We should probably print some very compact representation of all items. The exact item is not all that useful, but the module_ids on it are.